tests/Testdata/data-sources.md

```diff
@@ -0,0 +1,22 @@
+# Data Sources
+
+| Data Source         | Dataset name                      | Format    | License                            |
+|---------------------|-----------------------------------|-----------|------------------------------------|
+| [sf]                | nc                                | gpkg      | COPYRIGHT HOLDER: Edzer Pebesma    |
+| [Geosoftware2]      | clc_1000_PT                       | gml       | MIT                                |
```

Comment:
Where did you get the license information from? The project `carobro/Geosoftware2` is MIT, but the data files are _not_ created by them, just republished. I cannot believe they are all MIT originally.

Reply:
I created an issue requesting data source of files use in test.

Reply:
Done

Reply:
https://github.com/carobro/Geosoftware2/issues/75